### ISSUE TITLE
png sequence problems

### ISSUE THREAD

**Issue by GamePIN741** [airbnb/lottie-web#2898]
Ok, so I'm trying to use the bodymovin plugin to export a png sequence as a .json file for website integration with protopie, and after a long convoluted road that involved many many hours of hopeless research I thought I had found the solution with bodymovin, but when I render the image sequence there are frames with errors that just have the .png file indicator on them instead of the image, i've rendered this probably a hundred times and the errors seem to be in random places each time, they persist between platforms and I've looked and looked for hours and hours and hours to find a solution, but every post is just the same noobs having the same problem because they didn't check a box literally every tutorial tells you to check, so I couldn't find a solution, which brings me here.
so tl:dr, random frame errors, no online solutions I can find, help me
After Effects 2023, newest version of bodymovin, idk, my adobe subscription ran out and it needs to be renewed, windows 10
![image](https://user-images.githubusercontent.com/116318593/209416899-b4c0d77d-cf96-4822-862e-ac968a22d34f.png)
[Untitled Project.zip](https://github.com/airbnb/lottie-web/files/10297838/Untitled.Project.zip)

**Comment by bodymovin** [comment]
hi, it seems you've shared only the project but not the assets needed.
Do you have a link with the lottie rendered animation that I can check?

**Comment by GamePIN741** [comment]
yea, I can upload it
[https://drive.google.com/drive/folders/1oxRUWkG9NircaqwHGL7MvIb2mFk553q7?usp=sharing](url)
have to use google drive, because github has a 25mb limit, and its a 45mb file

**Comment by bodymovin** [comment]
thanks, do you have a link with the implementation running to check it out?

**Comment by GamePIN741** [comment]
got it running in protopie cloud
[https://cloud.protopie.io/p/eee72aad83f8c0644b00bf45](url)

**Comment by bodymovin** [comment]
it does look like there are some images missing from the render. The drive folder you shared seems to be incomplete also, can you make sure all the files are there?

**Comment by d-korolkov** [comment]
I can confirm that there is a problem with rendering long png sequence, let say 90 unique files.
Win, AE23, BM 5.12.2, include assets with compression. BM 5.10 gave the same problem.
After exporting, some random frames/images are missed (from 0 to about 10) inside JSON.
Only one solution works good:  compress files out of AE and copy PNGs as original assets checkbox - no one missed.

On macOS, all files are exporting and including, but compression works much worse the on Win.  Standard 80 compression gives about +300% of original compressed files. So better to compress files out of AE and copy PNGs as original assets checkbox.

BTW is there a way to control/setup __HIDDEN X-Factor 8 Premul_ export method?
It has just wonderful compression abilities on Windows, doesn't work good on macOs, so interesting to play with it somehow.

**Comment by Schmulek** [comment]
Did you fix the issue?
